src/browser/modules/D3Visualization/components/Explorer.tsx

```diff
@@ -129,7 +127,8 @@ export class ExplorerComponent extends Component<
       selectedItem,
       hoveredItem: selectedItem,
       freezeLegend: false,
-      width: defaultPanelWidth()
+      width: defaultPanelWidth(),
+      nodePropertiesExpanded: true
```

Comment:
Lifted this state so that the offset for the zoombuttons can be set to zero when the panel is contracted. The offset for the zoombuttons wouldn't be needed if we refactored the component tree, but that was out of scope